File: app/src/main/java/com/mnidersoft/movieclient/util/AppUtil.java
package com.mnidersoft.movieclient.util;

import java.util.Collection;

/**
 * Created by Allan.Menezes on 11/11/17.
 */

public class AppUtil {

    public static boolean isNullOrEmpty(Collection<?> objects) {
        return objects == null || objects.isEmpty();
    }
}

File: app/src/main/java/com/mnidersoft/movieclient/util/AppUtil.kt
@file:JvmName("AppUtil")

package com.mnidersoft.movieclient.util

import java.io.File

/**
 * Created by Allan.Menezes on 11/11/17.
 */

fun isNullOrEmpty(objects: Collection<*>?): Boolean {
    return objects?.isEmpty() ?: true
}

fun isNullOrEmpty(string: String?): Boolean {
    return string?.isEmpty() ?: true
}

fun getBasePath(): String {
    var basePath = "./src/"
    val file = File(basePath)
    if (!file.exists()) basePath = "./app/src/"
    return basePath
}